src/ApiGateways/Web.Bff.Shopping/aggregator/Program.cs

```diff
@@ -1,24 +1,225 @@
-await ﻿BuildWebHost(args).RunAsync();
+﻿var appName = "Ordering.API";
+var builder = WebApplication.CreateBuilder(args);
+builder.WebHost.ConfigureAppConfiguration(cb =>
+{
+    var sources = cb.Sources;
+    sources.Insert(3, new Microsoft.Extensions.Configuration.Json.JsonConfigurationSource()
```

Comment:
This is extremely weird. Hard no on this approach

Reply:
May be we can remove this section?